### PR TITLE
change barrnap filtering

### PR DESCRIPTION
**Pull request by d4straub**
Closes https://github.com/nf-core/ampliseq/issues/547

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by erikrikarddaniel — APPROVED** [review]
What barrnap calls "kingdoms" are actually domains (except "mito" of course which is neither). I really don't know what's best here: Use the same terminology as the tool or correct? I've made some suggestions that uses both terms. Do as you see fit best. Note however that there are a couple of spelling mistake corrections as well among my suggestions.

Otherwise, LGTM -- that was quick! :-)

File: nextflow_schema.json [hunk 1]
```diff
@@ -307,7 +307,7 @@
             "properties": {
                 "filter_ssu": {
                     "type": "string",
-                    "description": "Enable SSU filtering. Comma separated list of kingdoms in Barrnap.",
+                    "description": "Enable SSU filtering. Comma separated list of kingdoms in Barrnap. ASVs that have their lowest evalue in that kingdoms are chosen.",
```
[the remark this answers is not in the document]
Comment (d4straub) [reply nextflow_schema.json:310]: possible values are in https://nf-co.re/ampliseq/2.4.1/parameters#filter_ssu -> klick "options" which come from https://github.com/nf-core/ampliseq/blob/83877d2fe539293818be84dec3bd1f43238c9c3c/nextflow_schema.json#L309-L324.
But I could add something like "a combination of 'bac', 'arc', 'mito', 'euk'."?

[the remark this answers is not in the document]
Comment (d4straub) [reply nextflow_schema.json]: I did add such an info below, thanks!

[the remark this answers is not in the document]
Comment (erikrikarddaniel) [reply nextflow_schema.json]: Great!

**Pull request merged**